src/domain/functions/answerCorrectness/answersCorrectness.test.js

```diff
@@ -0,0 +1,19 @@
+const answerCorrectness = require('./answersCorrectness')
+
+
+test('Properly checks if answers is correct',()=>{
+    const question={
+"image": "base64imagestring",
+"answers": ["Luke Skywalker", "Leia Organa", "Darth Vader", "Darth Sidious"],
+"rightAnswer": "Darth Vader"
+};
+const answer="Darth Vader";
+const answer2="Darth Vade";
+const answer3="";
+    
+    expect(answerCorrectness(question,answer)).toBe(1);
+    expect(answerCorrectness(question,answer2)).toBe(0);
+    expect(answerCorrectness(question,answer3)).toBe(0);
+    
+})
```

Comment:
```suggestion

describe('Validate answers', () => {
it('returns true when answer is correct', () => {
const question={
"image": "base64imagestring",
"answers": ["Luke Skywalker", "Leia Organa", "Darth Vader", "Darth Sidious"],
"rightAnswer": "Darth Vader"
};
const answer="Darth Vader";
expect(isCorrectAnswer(question.rightAnswer,answer)).toBeTruthy();
})
}) 

```

I podobny test na wartość nieprawdziwą